cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -1,23 +1,36 @@
 import TopMenu from '../../support/fragments/topMenu';
 import Actions from '../../support/fragments/inventory/actions';
+import InventorySearchPanel from '../../support/fragments/inventory/inventorySearchPanel';
 
 describe('ui-inventory: actions', () => {
-  beforeEach('navigates to Actions', () => {
+  beforeEach('navigates to actions', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
     cy.visit(TopMenu.inventoryPath);
   });
 
   it('C196752 verifies action menu options before any search is conducted', () => {
-    const optionsShouldDisabled = [
-      Actions.saveUUIDOption,
-      Actions.saveCQLQueryOption,
-      Actions.exportMARCOption,
-      Actions.showSelectedRecordsOption
-    ];
+    cy.do(Actions.open());
 
-    Actions.open()
-      .then(() => {
-        optionsShouldDisabled.forEach(option => Actions.optionIsDisabled(option, true));
-      });
+    cy.expect([
+      Actions.saveUUIDsOption().is({ disabled: true }),
+      Actions.saveCQLQueryOption().is({ disabled: true }),
+      Actions.exportMARCOption().is({ disabled: true }),
+      Actions.showSelectedRecordsOption().is({ disabled: true }),
+    ]);
+  });
+
+  it('C196753 verifies action menu options after searching and selecting result', () => {
+    cy.do([
+      InventorySearchPanel.effectiveLocationInput.select([InventorySearchPanel.effectiveLocationValues.mainLibrary]),
```

Comment:
lets wrap this to  InventorySearchPanel.searchByEffectiveLocation(specialLocation)? And you can specify mainLibrary as default parameter value in this test. We will get InventorySearchPanel.searchByEffectiveLocation();
May be we can skip a few words in naming and get just InventorySearch.byEffectiveLocation();

Reply:
Yes, naming so bad. I will think how it can be fixed